<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gamewindow</class>
 <widget class="QMainWindow" name="gamewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>890</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>711</width>
      <height>871</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdgametime">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>120</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="ourmoney">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>680</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="building1_hp">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>680</y>
      <width>118</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="buliding2_hp">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>118</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="enemymoney">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>110</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="ourtower_left">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>590</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="ourtower_right">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>590</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="enemytower_right">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>210</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="enemytower_left">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="card1_button">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>810</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>card1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="card2_button">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>810</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>card2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="launch">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>810</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>launch</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="magic_time">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>810</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="hero_put">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>790</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>hero</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="killtime">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>820</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="majorkill">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>820</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>kill</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>712</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
